collect_app/src/main/java/org/odk/collect/android/activities/FormEntryActivity.java

```diff
@@ -705,8 +705,14 @@ public void run() {
                 if (filePath != null) {
                     new File(filePath).delete();
                 }
-                getContentResolver().delete(mediaUri, null, null);
+                try {
+                    getContentResolver().delete(mediaUri, null, null);
+                } catch (Exception e) {
+                    Toast.makeText(this,e.getMessage(),Toast.LENGTH_SHORT).show();
```

Comment:
This error doesn't actually affect the user (it just means the recorded file isn't removed) so no error should be shown. I think the right thing to do here is just `Timber.e(e)` to get a sense of how often it happens until we fix the root problem.